src/native/src/core/nova_renderer.h

```diff
@@ -7,12 +7,13 @@
 
 #include <memory>
 
-#include "../interfaces/iwindow.h"
+#include "interfaces/iwindow.h"
+#include "mc/mc_objects.h"
+
 #include "nova.h"
-#include "../mc/mc_objects.h"
+#include "types.h"
```

Comment:
I guess i probably just though the includes from the current directory should be together?